app/src/main/java/com/nutomic/syncthingandroid/service/RestApi.java

```diff
@@ -241,6 +253,7 @@ public void editFolder(Folder newFolder) {
 
     public void removeFolder(String id) {
         removeFolderInternal(id);
+        mCompletion.removeFolder(id);
```

Comment:
We don't need this, the ConfigSaved would handle this already.

Reply:
removed.